### PR TITLE
🐞 fix: hides dates from previous/next month in view of month

### PR DESCRIPTION
**Pull request by jorul**
## Proposed Changes

-

## Types of Changes

- [ ] New feature
- [x] Bug fix
- [ ] Enhancement/optimization
- [ ] Refactor
- [ ] Style
- [ ] Build/dependencies
- [ ] Other

## Issue(s) fixed or closed by this PR

-

## Checklist

- [ ] I have added tests to cover my changes (for features/bugs)
- [x] I am pleased with the readability of the code (if not, state where you need input)
- [x] I have tested the changes and verified that they work and do not break anything (to the best of my ability)

from:
image

to:
image

### CODE REVIEW

File: frontend/src/components/Calendar/CalendarDay.tsx [hunk 1]
```diff
@@ -48,13 +48,13 @@ const useStyles = makeStyles((theme: Theme) => ({
 
 const CalendarDay: React.VFC<Props> = (props) => {
```
Comment (larwaa) [inline frontend/src/components/Calendar/CalendarDay.tsx:49]:
```suggestion
const CalendarDay: React.VFC<Props> = ({ onClick, value, isHidden }) => {
```

File: frontend/src/components/Calendar/CalendarDay.tsx [hunk 2]
```diff
@@ -48,13 +48,13 @@ const useStyles = makeStyles((theme: Theme) => ({
...
   const classes = useStyles(props);
-  const { onClick, value } = props;
+  const { onClick, value, isHidden } = props;
```
Comment (larwaa) [inline frontend/src/components/Calendar/CalendarDay.tsx:51]:
```suggestion
```

File: frontend/src/components/Calendar/index.tsx [hunk 3]
```diff
@@ -99,6 +99,11 @@ const Calendar: React.FC<CalendarProps> = ({
     );
   };
 
+  const isHidden = (date: dayjs.Dayjs, month: dayjs.Dayjs) => {
+    console.log(date.month() != month.month());
```
Comment (larwaa) [inline frontend/src/components/Calendar/index.tsx:103]:
```suggestion
```

**Review by larwaa — CHANGES_REQUESTED** [review]
Minor stuff

File: frontend/src/components/Calendar/index.tsx (hunk 3 above)
Comment (Jorgro) [inline frontend/src/components/Calendar/index.tsx:103]: 🕵️‍♀️

**Review by Jorgro — CHANGES_REQUESTED** [review]
🤭

File: frontend/src/components/Calendar/CalendarDay.tsx (hunk 1 above)
Comment (jorul) [reply frontend/src/components/Calendar/CalendarDay.tsx:49]: nope, props is used for useStyles in line 50

**Review by larwaa — APPROVED** [review]
LGTM 🥳

**Review by MorganFH — APPROVED** [review]
LGTM

**Pull request merged**